File: ESTUDO DE LINGUAGENS/JAVASCRIPT/Questões uri/1010.js
let pegarValores = (line) => line.split(" ").map(a => +a)
let [a,b,c] = pegarValores(lines.shift())

let pegarValoress = (line) => line.split(" ").map(a => +a)
let [a1,b1,c1] = pegarValoress(lines.shift())

let soma=((b*c)+(b1*c1))
console.log(`VALOR A PAGAR: R$ ${soma.toFixed(2)}`)

File: ESTUDO DE LINGUAGENS/C/Questões Uri/1010.c
#include <stdio.h>
 
int main() {
	int cod, cod1,qtd,qtd1;
	double valor,valor1,soma;
	scanf("%i %i %lf",&cod,&qtd,&valor);
	
	scanf("%i %i %lf",&cod1,&qtd1,&valor1);
	soma=(qtd*valor)+(qtd1*valor1);
    printf("VALOR A PAGAR: R$ %.2f\n",soma);
    return 0;
}